deeposlandia/inference.py

```diff
@@ -48,7 +48,8 @@ def add_program_arguments(parser):
     """
     parser.add_argument('-D', '--dataset',
                         required=True,
-                        help="Dataset type (either mapillary, shapes or aerial)")
+                        help=("Dataset type (either mapillary, shapes, aerial"
```

Comment:
you pass a specific list of choices with argparse. See https://docs.python.org/3/library/argparse.html#choices

Reply:
Modified. Thanks for this suggestion!